syntax = "proto3";
import "detection_types.proto";

service AnomalyDetection {
  rpc TwitterAnomalies (stream anomalydetection.Request) returns (stream anomalydetection.Response) {}
}
